server/gql/models/purchasedProducts/customCreateResolver.js

```diff
@@ -23,10 +25,22 @@ export const updateRedis = async res => {
     })
   );
 };
+
+export const publishMessage = async (args, res) => {
+  pubsub.publish(SUBSCRIPTION_TOPICS.NOTIFICATIONS, {
```

Comment:
let's have a more intuitive topicName. Something that reflects that this is associated to products of a store